### DISCUSSION TITLE
Holds, top of descent and top of climb indicators…

### DISCUSSION THREAD

**Discussion by TLSY2** [flybywiresim/aircraft#5291] [Ideas]
Hi,

The hold option, i dont think its a lot of work, i think its quite simple to make and working title already has it.

And as for the indicators on the nd, you had them before, i dont know what happens but working title already has it.

Thanks!

**Comment by tracernz** [comment]
A complete Airbus implementation of these things is more complex than what WT has (or are trying to replicate from Garmin). These things are in progress.

**Reply by TLSY2** [reply]
The hold isnt that complicated as you can make it manually chose a track and time... Or at least i dont think it is complicatef... I wish i knew how to code, i would be making these myself...

**Reply by tracernz** [reply]
How about the transition into the hold? That is one of the most complicated pieces of the Airbus LNAV algorithms. ;)

**Reply by TLSY2** [reply]
> How about the transition into the hold? That is one of the most complicated pieces of the Airbus LNAV algorithms. ;)

you enter it at a certain waypoint, im not saying do it like the real plane, just a simple option... Like enter manually...

**Reply by ExampleWasTaken** [reply]
The whole point of the project is to be as close to the real aircraft as possible, so you probably gonna have to wait a little longer for it to be implemented.

**Comment by hotshotp** [comment]
"The hold option, i dont think its a lot of work, i think its quite simple to make and working title already has it." it is actually a lot of work. please do not make comments like these. our developers have been tirelessly working on an LNAV implementation for weeks and do not need comments like these to stress them out more. As for top of climb/descent that will come later after LNAV with a separate VNAV implementation.

**Reply by TLSY2** [reply]
> "The hold option, i dont think its a lot of work, i think its quite simple to make and working title already has it." it is actually a lot of work. please do not make comments like these. our developers have been tirelessly working on an LNAV implementation for weeks and do not need comments like these to stress them out more. As for top of climb/descent that will come later after LNAV with a separate VNAV implementation.

calm down, they are not working full time, they have no stress as they do it for fun. I think its easy because of the logic, now tell me if im wrong. If you add a basic one for example at waypoint WAXIS add hold then select track and time, thats all... I dont know what complications that adds.... I dont know like i keep saying i dont know code and if you can explain that would be great.

**Reply by hotshotp** [reply]
> > "The hold option, i dont think its a lot of work, i think its quite simple to make and working title already has it." it is actually a lot of work. please do not make comments like these. our developers have been tirelessly working on an LNAV implementation for weeks and do not need comments like these to stress them out more. As for top of climb/descent that will come later after LNAV with a separate VNAV implementation.
> 
> calm down, they are not working full time, they have no stress as they do it for fun. I think its easy because of the logic, now tell me if im wrong. If you add a basic one for example at waypoint WAXIS add hold then select track and time, thats all... I dont know what complications that adds.... I dont know like i keep saying i dont know code and if you can explain that would be great.

A lot of the devs take this project seriously and therefore do get stressed out over it. I won't explain to you the details but there definitely is a lot of work involved in this.

Now you say that you think it's easy because of the logic, and I say you are not correct. You are just simply wrong. There are many factors involved in how different holds behave, how it is transitioned into, how the fmgc reacts to it, and so on.

**Comment by TLSY2** [comment]
Fine, I agree but I dont agree with the stressed modders, like if they dont want to do it they dont have to, I dont think they get stressed, i think they get excited... Anyway they dont receive enough fame, for every implementation should be a tweet with the guys @ showing off how much he done, that would bring happiness.

**Reply by Benjozork** [reply]
Ok then you're more than welcome to come help out

**Reply by TLSY2** [reply]
> then you're more than welcome to come help out

exactly, but i dont know what im good for this org... I can make videos and edit them.

**Reply by AussiePilot123** [reply]
Dude! These devs are putting their time and effort into making this mod. They don't need you trying to push them along. The hold feature will come in due time. Just fly the holds manually until then, it's not that hard.
